### PR TITLE
docs: stream operator throttle typo

### PR DESCRIPTION
**Pull request by Roiocam**
found something typo and missing by reviewing in resolves https://github.com/apache/pekko-persistence-jdbc/pull/199

### CODE REVIEW

**Review by He-Pin — APPROVED** [review]
lgtm

File: docs/src/main/paradox/stream/operators/Source-or-Flow/throttle.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Limit the throughput to a specific number of elements per time unit, or a specif
 
 ## Description
 
-Limit the throughput to a specific number of elements per time unit, or a specific total cost per time unit, where
+Limit the throughput to a specific number of elements per time unit, or a specific total cost per-time unit, where
```
Comment (raboof) [inline docs/src/main/paradox/stream/operators/Source-or-Flow/throttle.md:14]: I think it really is "cost per time unit", no idea what "cost per-time unit" would mean

File: docs/src/main/paradox/stream/operators/Source-or-Flow/throttle.md [hunk 2]
```diff
@@ -41,8 +41,8 @@ Scala
 Java
 :   @@snip [Throttle.java](/docs/src/test/java/jdocs/stream/operators/sourceorflow/Throttle.java) { #throttle-with-burst }
 
-The extra argument to set the `ThrottleMode` to `shaping` tells `throttle` to make pauses to avoid exceeding 
-the maximum rate. Alternatively we could set the throttling mode to cause a stream failure when upstream is faster
+The extra argument to set the `ThrottleMode` to `shaping` tells throttle to make pauses to avoid exceeding 
+the maximum rate. Alternatively we could set the `ThrottleMode` to `enforcing` cause a stream failure when upstream is faster
```
Comment (raboof) [inline docs/src/main/paradox/stream/operators/Source-or-Flow/throttle.md:45]:
```suggestion
the maximum rate. Alternatively we could set the `ThrottleMode` to `enforcing` to cause a stream failure when upstream is faster
```

File: docs/src/main/paradox/stream/operators/Source-or-Flow/throttle.md (hunk 1 above)
Comment (Roiocam) [reply docs/src/main/paradox/stream/operators/Source-or-Flow/throttle.md:14]:
```suggestion
Limit the throughput to a specific number of elements per time unit, or a specific total cost per time unit, where
```

**Review by raboof — APPROVED** [review]

**Comment by raboof** [comment]
failure was the link-validator timing out connecting to http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html . Rebuilding, #1309

**Pull request merged**